content/community.html

```diff
@@ -52,6 +52,17 @@ <h1>Contributing</h1>
       <a href="https://groups.google.com/forum/#!forum/prometheus-developers">prometheus-developers</a> – for discussions around Prometheus development
     </p>
 
+    <h1>Development summits</h1>
+    <p>
+      We strive to be as open as public as public. Technical discussions happen on the development list, our in-person meeting notes are public, and we have public calls.
+    </p>
+    <p>
+      <a href=""https://docs.google.com/document/d/1DaHFao0saZ3MDt9yuuxLaCQg8WGadO8s44i3cxSARcM/edit#heading=h.bbvw42ndm0nb>2017 dev summit notes</a>
+    </p>
+    <p>
+      <a href=""https://docs.google.com/document/d/1-C5PycocOZEVIPrmM1hn8fBelShqtqiAmFptoG4yK70/edit>2018 dev summit notes</a>
```

Comment:
These links are lacking a bit of context as to what those dev summits are, who was (and in the future, is) allowed/invited to attend, and why.